diff --git a/node_modules/@types/chrome/index.d.ts b/node_modules/@types/chrome/index.d.ts
index 2b0d9e6..e6f7b97 100644
--- a/node_modules/@types/chrome/index.d.ts
+++ b/node_modules/@types/chrome/index.d.ts
@@ -311,6 +311,14 @@ declare namespace chrome.action {
      */
     export function setBadgeText(details: BadgeTextDetails, callback?: () => void): void;
 
+    /**
+     * Since Chrome 88.
+     * Sets the icon for the action. The icon can be specified either as the path to an image file or as the pixel data from a canvas element,
+     * or as dictionary of either one of those. Either the path or the imageData property must be specified.
+     * @return The `setIcon` method provides its result via callback or returned as a `Promise` (MV3 only). Since Chrome 96.
+     */
+    export function setIcon(details: TabIconDetails): Promise<void>;
+
     /**
      * Since Chrome 88.
      * Sets the icon for the action. The icon can be specified either as the path to an image file or as the pixel data from a canvas element,
